api/types/saml.go

```diff
@@ -134,14 +134,18 @@ func (o *SAMLConnectorV2) SetResourceID(id int64) {
 
 // WithoutSecrets returns an instance of resource without secrets.
 func (o *SAMLConnectorV2) WithoutSecrets() Resource {
-	k := o.GetSigningKeyPair()
-	if k == nil {
+	k1 := o.GetSigningKeyPair()
+	k2 := o.GetEncryptionKeyPair()
+	if k1 == nil && k2 == nil {
```

Comment:
If only the encryption key pair was not set, this would lead to `nil` dereference, wouldn't it?

Reply:
Yep, you're correct. Fixed